src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -228,6 +262,9 @@ export function makeGeneticTrackDatum_Data_Type2(oncoprinterInputLine:Oncoprinte
         // these are the same always or almost always
         hugoGeneSymbol:oncoprinterInputLine.hugoGeneSymbol,
         proteinChange:oncoprinterInputLine.proteinChange,
+        mutationStatus: oncoprinterInputLine.isGermline ? "germline": "",
```

Comment:
seems likely there might already be constants somewhere for these (germline and Putative_Driver)